### PR TITLE
[26705] Unit test cases for ReadwriteSplittingRuleChangedEventCreator

### LINKED ISSUE #26705: Add unit test for ReadwriteSplittingRuleChangedEventCreator

### PR DESCRIPTION
**Pull request by surivig18**
Fixes #26705

Changes proposed in this pull request:
  -

---

Before committing this PR, I'm sure that I have checked the following options:
- [✅] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [ ✅] I have self-reviewed the commit code.
- [ ✅] I have (or in comment I request) added corresponding labels for the pull request.
- [✅ ] I have passed maven check locally : `./mvnw clean install -B -T1C -Dmaven.javadoc.skip -Dmaven.jacoco.skip -e`.
- [✅ ] I have made corresponding changes to the documentation.
- [ ✅] I have added corresponding unit tests for my changes.

### CODE REVIEW

File: features/readwrite-splitting/core/src/test/java/org/apache/shardingsphere/readwritesplitting/event/ReadwriteSplittingRuleChangedEventCreatorTest.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                ReadwriteSplittingRuleNodePathProvider.LOAD_BALANCERS, "test"), instanceOf(DeleteReadwriteSplittingLoadBalancerEvent.class));
+        
+        assertThat(readwriteSplittingRuleChangedEventCreator.create("test",
+                new DataChangedEvent("RANDOM", "RANDOM", DataChangedEvent.Type.ADDED),
+                ReadwriteSplittingRuleNodePathProvider.LOAD_BALANCERS, "test"), instanceOf(AlterReadwriteSplittingLoadBalancerEvent.class));
+    }
+    
+    @Test
+    void assertReadWriteEventsRuleChangeCreatorThrowsUnsupportedException() {
+
```
Comment (zhaojinchao95) [inline features/readwrite-splitting/core/src/test/java/org/apache/shardingsphere/readwritesplitting/event/ReadwriteSplittingRuleChangedEventCreatorTest.java:70]: Remove empty line please.

File: features/readwrite-splitting/core/src/test/java/org/apache/shardingsphere/readwritesplitting/event/ReadwriteSplittingRuleChangedEventCreatorTest.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        readwriteSplittingRuleChangedEventCreator = new ReadwriteSplittingRuleChangedEventCreator();
+    }
+    
+    @Test
+    void assertReadWriteSplittingRuleChangeEventForDataSource() {
+        
+        assertThat(readwriteSplittingRuleChangedEventCreator.create("test",
+                new DataChangedEvent("RANDOM", "RANDOM", DataChangedEvent.Type.DELETED),
+                ReadwriteSplittingRuleNodePathProvider.DATA_SOURCES, "test"), instanceOf(DeleteReadwriteSplittingDataSourceEvent.class));
+
```
Comment (zhaojinchao95) [inline features/readwrite-splitting/core/src/test/java/org/apache/shardingsphere/readwritesplitting/event/ReadwriteSplittingRuleChangedEventCreatorTest.java:50]: Remove empty line please.

File: features/readwrite-splitting/core/src/test/java/org/apache/shardingsphere/readwritesplitting/event/ReadwriteSplittingRuleChangedEventCreatorTest.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+                ReadwriteSplittingRuleNodePathProvider.DATA_SOURCES, "test"), instanceOf(AddReadwriteSplittingDataSourceEvent.class));
+        
+    }
+    
+    @Test
+    void assertReadWriteEventsRuleChangeCreatorForLoadBalancers() {
+        assertThat(readwriteSplittingRuleChangedEventCreator.create("test",
+                new DataChangedEvent("RANDOM", "RANDOM", DataChangedEvent.Type.DELETED),
+                ReadwriteSplittingRuleNodePathProvider.LOAD_BALANCERS, "test"), instanceOf(DeleteReadwriteSplittingLoadBalancerEvent.class));
+
```
Comment (zhaojinchao95) [inline features/readwrite-splitting/core/src/test/java/org/apache/shardingsphere/readwritesplitting/event/ReadwriteSplittingRuleChangedEventCreatorTest.java:62]: Remove empty line please.

File: features/readwrite-splitting/core/src/test/java/org/apache/shardingsphere/readwritesplitting/event/ReadwriteSplittingRuleChangedEventCreatorTest.java (hunk 1 above)
Comment (surivig18) [reply features/readwrite-splitting/core/src/test/java/org/apache/shardingsphere/readwritesplitting/event/ReadwriteSplittingRuleChangedEventCreatorTest.java]: Removed 👍

File: features/readwrite-splitting/core/src/test/java/org/apache/shardingsphere/readwritesplitting/event/ReadwriteSplittingRuleChangedEventCreatorTest.java (hunk 2 above)
Comment (surivig18) [reply features/readwrite-splitting/core/src/test/java/org/apache/shardingsphere/readwritesplitting/event/ReadwriteSplittingRuleChangedEventCreatorTest.java]: Removed 👍

File: features/readwrite-splitting/core/src/test/java/org/apache/shardingsphere/readwritesplitting/event/ReadwriteSplittingRuleChangedEventCreatorTest.java (hunk 3 above)
Comment (surivig18) [reply features/readwrite-splitting/core/src/test/java/org/apache/shardingsphere/readwritesplitting/event/ReadwriteSplittingRuleChangedEventCreatorTest.java]: Removed 👍

**Review by terrymanu — APPROVED** [review]

**Pull request merged**